assemblyline/odm/models/ontology/filetypes/macho.py

```diff
@@ -0,0 +1,77 @@
+from assemblyline import odm
+
+
+@odm.model(index=True, store=False)
+class MachO(odm.Model):
+    @odm.model(index=True, store=False)
+    class Header(odm.Model):
+        entrypoint = odm.Optional(odm.EmptyableKeyword(copyto="__text__"))
+        cpu_type = odm.Optional(odm.EmptyableKeyword(copyto="__text__"))
+        cpu_sub_type = odm.Optional(odm.Integer())
+        file_type = odm.Optional(odm.EmptyableKeyword(copyto="__text__"))
+        flags = odm.Optional(odm.EmptyableKeyword(copyto="__text__"))
+        numberof_loadcommands = odm.Optional(odm.Integer())
+        sizeof_loadcommands = odm.Optional(odm.Integer())
+        minimum_os_version = odm.Optional(odm.EmptyableKeyword(copyto="__text__"))
+        platform = odm.Optional(odm.EmptyableKeyword(copyto="__text__"))
+        sdk_version = odm.Optional(odm.EmptyableKeyword(copyto="__text__"))
+        linker = odm.Optional(odm.EmptyableKeyword(copyto="__text__"))
+        base = odm.Optional(odm.EmptyableKeyword(copyto="__text__"))
+        is_ios = odm.Optional(odm.Boolean())
+        is_macos = odm.Optional(odm.Boolean())
+        is_position_independent = odm.Optional(odm.Boolean())
+        is_norized = odm.Optional(odm.Boolean())
```

Comment:
Would you be able to point me toward some documentation for this field, and what it signifies in the MachO architecture?